scripts/solve_network.py

```diff
@@ -1421,6 +1421,7 @@ def solve_network(
     update_config_from_wildcards(snakemake.config, snakemake.wildcards)
 
     solve_opts = snakemake.params.solving["options"]
+    ember_hotfix= snakemake.params.ember_hotfix
```

Comment:
please use a different name than `ember_hotfix`, this is unclear and doesn't provide any information on what this actually does - both for the variable as well as the `params` setting